endpoints/groups.md

```diff
@@ -18,13 +18,28 @@
 {
   "id": 1,
   "name": "Group A",
+  "saved_message_id": "abcdefg",
+  "campaign_template_id": 1,
+  "profile_id": 1,
   "contacts_count": 1,
   "created_at": "2013-05-07T17:51:05Z",
   "updated_at": "2013-05-07T17:51:05Z",
   "contacts": "/groups/:id/contacts.json"
 }
 ```
 
+## Group Automation
+When the `saved_message_id` and/or `campaign_template_id` are set, each time a contact is added to
+the group they will automatically receive the specified Saved Message and/or Campaign. If the
+`profile_id` is not specified, the automated messages will be sent with the user's default profile
+that owns the group.
```

Comment:
Should we also add that you will be unable to add a profile if it is not owned by the user?

Reply:
I think that's covered just below this.